rnahybrid.gap

```diff
@@ -706,32 +705,33 @@ algebra alg_probing implements sig_rnahybrid(alphabet = char, answer = double) {
   }
 }
 
+
 /*
 This grammar has been extracted from src/hybrid_core.c of RNAhybrid-2.1.2.tar.gz by Stefan Janssen (2021-08-12)
 It seems to be equivalent to the Haskell Version https://bibiserv.cebitec.uni-bielefeld.de/cgi-bin/adp_RNAhybrid
 
 SMJ 2024-06-27: I've changed the grammar such that each candidate has "complete" as its root. This is useful to
-prepend the pretty print string with some left information, i.e. "target 5' " and "miRNA  3' "
+                prepend the pretty print string with some left information, i.e. "target 5' " and "miRNA  3' "
```

Comment:
I don't know if it's due to the display in Git but the indentation here is kind of messed up.

Reply:
should look like this
![image](https://github.com/jlab/fold-grammars/assets/11960616/a684318d-d9e7-4c1c-925f-35119a45dc54)
i.e. continue the comment left of the SMJ timestamp

Reply:
Well then Git is at fault here :D